src/Cameras/babylon.freeCamera.ts

```diff
@@ -104,14 +104,14 @@
         }
 
         // Collisions
-        private _collisionMask = -1;
+        private _collisionMask = 0;

```

Comment:
0 will always cause a collision when we're doing ((mask & group) === 0), it will act as no groups or masks has been set